cl/search/templates/includes/order_by_dropdown.html

```diff
@@ -59,6 +59,10 @@
         {% endif %}
 
         {% if v == SEARCH_TYPES.PEOPLE %}
+            <option value="score desc"
```

Comment:
The logic in this file is a bit more complicated than I think you realized. I think this change should be on line 11, no?

Reply:
yes I think it can actually be simplified a bit more